RegexFileSearcher/RegexFileSearcher/FileHandlerException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
```

Comment:
```suggestion
﻿using System;
```
Unnecessary `using` directives.